packages/wallets/src/AppContent.tsx

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
-import { DesktopWalletsList } from './components';
-// import WalletsCarousel from './components/WalletCarousel';
+import { DesktopWalletsList, WalletsCarousel } from './components';
 import './AppContent.scss';
 
 const AppContent: React.FC = () => {
+    const mobile_width = 768;
+
+    //Temporary solution until we have a proper mobile view
+    const is_mobile = window.innerWidth <= mobile_width;
```

Comment:
```suggestion
    const is_mobile = useMobile();
```
Needs to be updated when farhan's pr is merged